code/packaging/app-config/src/main/resources/META-INF/cattle/system-services/defaults.properties

```diff
@@ -69,6 +69,8 @@ metadata.increment.processes=account.update, \
     hostipaddressmap.activate, \
     host.remove, \
     host.update, \
+    host.activate, \
+    host.deactivate, \
     instancehostmap.activate, \
```

Comment:
@alena1108 any bad repercussions from adding host activate and decactivate to metadata.increment.process?

Reply:
@cjellick not really. We already have plenty of processes triggering meta update...hard to make it worse. Plus activate/deactivate doesn't happen too often.